blog about monitoring deprecation (#925)
kyma-project/website: content/blog-posts/2022-12-09-monitoring-deprecation/index.md

- Security: Your historical metrics are exposed to a potential attacker of the cluster and with that, cannot be used for forensic analysis.
- Operations: If the cluster is not accessible for any reason, you cannot access the current and historical metrics for troubleshooting.

Another factor that influenced our decision is that Grafana Labs changed the licensing for Grafana to AGPLv3. This made us rethink the observability capabilities of Kyma overall, reinforcing our strategy shift towards integration into existing observability systems over in-cluster managing of such systems. For the same reason, we accelerated the execution of the strategy and, with that, the deprecation of the Monitoring component.

Overall, we no longer believe that a managed in-cluster solution meets the typical requirements for production workloads, so we won't invest further into that. Instead, Kyma is going to support you with the integration into central systems, ideally also providing solutions that are part of the SAP BTP offering.

## Impact
For now, nothing has changed as it is a deprecation notice only. However, in half a year or later:

> - We will remove the Prometheus and Grafana setup in it's current shape and with that, the ability to browse your metrics.

Reviewer: ```suggestion
- We will remove the Prometheus and Grafana setup in its current shape, and with that, you will lose the ability to browse your metrics.
```

Still, you have the following options:

> - You can manually install the classic in-cluster monitoring solution. Read [Installing a custom Prometheus stack(https://github.com/kyma-project/examples/tree/main/prometheus) in Kyma for more details.

Reviewer: ```suggestion
- You can manually install the classic in-cluster monitoring solution. Read [Installing a custom Prometheus stack](https://github.com/kyma-project/examples/tree/main/prometheus) in Kyma for more details.
```

- You can continue bringing your custom solution for the collection and long-term storage of metrics inside or outside the cluster.

> You can continue integrating with commercial monitoring offerings as outlined in an SAP specific mission to [Extend a Kyma Based Multitenant Application With Day 2 Operations](https://discovery-center.cloud.sap/missiondetail/3999/).

Reviewer: ```suggestion
You can continue integrating with commercial monitoring offerings as outlined in an SAP-specific mission to [Extend a Kyma Based Multitenant Application With Day 2 Operations](https://discovery-center.cloud.sap/missiondetail/3999/).
```

> You can continue integrating with commercial monitoring offerings as outlined in an SAP-specific mission to [Extend a Kyma Based Multitenant Application With Day 2 Operations](https://discovery-center.cloud.sap/missiondetail/3999/).

Reviewer: ```suggestion
- You can continue integrating with commercial monitoring offerings as outlined in an SAP-specific mission to [Extend a Kyma Based Multitenant Application With Day 2 Operations](https://discovery-center.cloud.sap/missiondetail/3999/).
```
